README.md

```diff
@@ -49,3 +49,23 @@ To generate a personal token with the necessary permissions, please visit [Creat
 - [twitter](https://twitter.com/yashrajbasan2)
 - [instagram](https://www.instagram.com/yashraj.dev/)
 - [linkedin](https://www.linkedin.com/in/yashraj-basan-11b915157/)
+
+## Issues
+
+If something doesn’t work, please [file an issue](https://github.com/yashrajb/devtrackr-vscode-extension/issues/new).
+
+## Branches and contributions
```

Comment:
If anyone wants to contribute, they should first fork the repository and then, after making changes, create a pull request. Also, please change the name of this section to 'Contribute.' It will be helpful if you list down the steps for contribution.

Reply:
Done! Check if it fits the requirements. Thanks for the suggestions of code review